modules/contentbox/models/ui/ThemeService.cfc

```diff
@@ -664,16 +664,32 @@ component accessors="true" threadSafe singleton{
 							<div class="panel-body">
 					' );
 
+					// Show group intro if there is one
+					if( structKeyExists( thisSettingMD, "groupIntro" ) and thisSettingMD.groupIntro gt '' ){
+						writeOutput( '<div style="padding:5px 0px 5px;border-bottom: 1px solid ##ddd;margin-bottom:20px;">' & thisSettingMD.groupIntro & '</div>' );
```

Comment:
This was mainly for testing. If we're happy, I added a couple of utility classes and a specific theme class to the css.